src/cryptography/x509/oid.py

```diff
@@ -61,7 +61,15 @@ def __hash__(self):
 
     @property
     def _name(self):
-        return _OID_NAMES.get(self, "Unknown OID")
+        return _OID_NAMES.get(self, ("Unknown OID", None))[0]
+
+    @property
+    def longname(self):
```

Comment:
When I suggested adding `name`, the maintainers wanted it to be `_name` so as not to commit to supporting these convenience properties in the long term. I suspect they will want the same here.